### ISSUE TITLE
Synthesis with other person out of RAVDESS

### ISSUE THREAD

**Issue by hathubkhn** [keonlee9420/Cross-Speaker-Emotion-Transfer#11]
Hello author, 
Firstly, thank you for giving this repo, it is really nice.
I have a question that:
1. I download CMU data with single person with 100 audios and make speaker embedding vector and synthesis with this, the performance is not good. I cannot detect any words.
2. Should we need to fine-tuning deep-speaker model to generate speaker embedding with my data.

Thank you

**Comment by keonlee9420** [comment]
Hi @hathubkhn , thanks for your attention.
1. There could be a various reason for such case. Could you please share the tensorboard logs and some samples audio with mel-spectrogram?
2. It might be, but it depends on the number of speakers and their features.

**Comment by hathubkhn** [comment]
Hi, 
1. When waiting for your response, I try to finetune in LJSPEECH data and I can synthesize the sentence but it is not high quality. I will attach my Mel-spectrogram below and please help me to find out how to improve
2. I want to use your repo to make voice cloning, I am not sure it cannot, so that based on yourTTS I make another loss for speaker similarity. And training from scratch. Is it possible?

**Comment by hathubkhn** [comment]
![kids_are_sitting_on_the_door_and_today_is_very_nice_I_want_to_go_out_Actor_001_neutral](https://user-images.githubusercontent.com/30932833/188032650-599100ac-b5ed-4241-8e06-bb4fb9da8bc7.png)

**Comment by hathubkhn** [comment]
Here is my training from scratch when I add speaker loss and train with LJspeech dataset
![Screenshot 2022-08-31 at 5 35 56 PM](https://user-images.githubusercontent.com/30932833/188032737-8609d489-2727-4cab-9fa6-95a248b427a4.png)

**Comment by hathubkhn** [comment]
Here is my training
![Screenshot 2022-09-02 at 9 09 47 AM](https://user-images.githubusercontent.com/30932833/188032851-d9947918-618d-49a9-8213-5dc15e22efd0.png)
 from scratch when adding speaker loss (SCL-speaker consistance loss) and training with LJSPEECH

**Comment by keonlee9420** [comment]
Ah, so sorry for the late response. I thought I replied to your comments.

1. It might be due to the light weight conv. Replacing it with normal transformer block will resolve the quality issue.
2. Yes, if the lambda (weight of each loss) is carefully assigned by some experiments.

**Comment by rsandx** [comment]
@keonlee9420,

About your last point 1 for a potential solution of the quality issue, can you provide an example for replacing the light weight conv. with normal transformer block? Thanks
